tools/arriba-drawing/2.3.0/arriba-drawing__2.3.0.cwl

```diff
@@ -0,0 +1,232 @@
+      prefix: --cytobands=
+      separate: false
+      position: 5
+  protein_domains:
+    label: protein domains
+    doc: |
+      GFF3 file containing the genomic coordinates of protein domains
+    type: File
+    inputBinding:
+      prefix: --proteinDomains=
+      separate: false
+      position: 6
+  sample_name:
+    label: sample name
+    doc: |
+      Sample name
+    type: string?
+    inputBinding:
+      prefix: --sampleName=
+      separate: false
+      position: 7
+  squish_introns:
+    label: squish introns
+    doc: |
+      Squish introns. Default true
+    type: boolean?
+    inputBinding:
+      prefix: --squishIntrons=
+      separate: false
+      position: 8
+  print_exon_labels:
+    label: print exon labels
+    doc: |
+      Print exon labels. Default true
+    type: boolean?
+    inputBinding:
+      prefix: --printExonLabels=
+      separate: false
+      position: 9
+  render_3d_effect:
+    label: render 3d effect
+    doc: |
+      Render 3d effect. Default true
+    type: boolean?
+    inputBinding:
+      prefix: --render3dEffect=
+      separate: false
+      position: 10
+  pdf_width:
+    label: pdf width
+    doc: |
+      PDF width. Default 11.692
+    type: int?
+    inputBinding:
+      prefix: --pdfWidth=
+      separate: false
+      position: 11
+  color_1:
+    label: color 1
+    doc: |
+      color 1. Default #e5a5a5
+    type: string?
+    inputBinding:
+      prefix: --color1=
+      separate: false
+      position: 12
+  color_2:
+    label: color 2
+    doc: |
+      color 2. Default #a7c4e5
+    type: string?
+    inputBinding:
+      prefix: --color2=
+      separate: false
+      position: 13
+  merge_domains_overlapping_by:
+    label: merge domains overlapping
+    doc: |
+      Merge domains overlapping. Default 0.9
+    type: int?
+    inputBinding:
+      prefix: --mergeDomainsOverlappingBy=
+      separate: false
+      position: 14
+  optimize_domain_colors:
+    label: optimize domain colors
+    doc: |
+      Optimize domain colors. Default false.
+    type: boolean?
+    inputBinding:
+      prefix: --optimizeDomainColors=
+      separate: false
+      position: 15
+  font_size:
+    label: font size
+    doc: |
+      Font size. Default 1.
+    type: int?
+    inputBinding:
+      prefix: --fontSize=
+      separate: false
+      position: 16
+  font_family:
+    label: font family
+    doc: |
+      Font family. Default Helvetica.
+    type: string?
+    inputBinding:
+      prefix: --fontFamily=
+      separate: false
+      position: 17
+  transcript_selection:
+    label: transcript selection
+    doc: |
+      Transcript selection. Default coverage
+    type: string?
+    inputBinding:
+      prefix: --transcriptSelection=
+      separate: false
+      position: 18
+  fixed_scale:
+    label: fixed scale
+    doc: |
+      Fixed scale. Default 0.
+    type: int?
+    inputBinding:
+      prefix: --fixedScale=
+      separate: false
+      position: 19
+  covergae_range:
```

Comment:
Should be coverage_range

Reply:
Few multiline editing tricks can prevent one from typing this out manually, quite a bit faster and reduces risk of making these errors, can discuss in our next 1-1